tck/src/main/java/org/eclipse/microprofile/lra/tck/participant/api/ContextTckResource.java

```diff
@@ -296,8 +295,7 @@ public Response compensateWork(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) URI lraId,
         return getEndPhaseResponse(false);
     }
 
-    // Complete methods SHOULD use @PUT but to verify that other HTTP methods are also supported use here HTTP POST
-    @POST
+    @PUT
     @Path("/complete")
     @Complete
     public Response completeWork(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) URI lraId,
```

Comment:
I can't comment on lines that haven't been changed, but you missed `@Status` method on line [313](https://github.com/eclipse/microprofile-lra/pull/198/files#diff-4c219933e42e7b38d324e42b907623a2R313) which is still defined with `@PUT` and not `@GET`

Reply:
Well spotted